### DISCUSSION TITLE
软件更新后一直Error: self check denied怎么办

### DISCUSSION THREAD

**Discussion by Mysterious-Blue** [natfrp/wiki#220] [使用问题] [answered]
在更新新版本frpc v0.42.0-sakura-5之后，隧道无法启动，一直提示Error: self check denied，然后就退出了，换隧道和节点也不行。

**Comment by fakeboboliu** [comment, marked as answer]
您好，此问题通常因为启动器文件损坏，请尝试卸载启动器重新安装。

**Reply by Mysterious-Blue** [reply]
好了，谢谢

**Comment by Lurito** [comment]
同样的问题。在官网下载 frpc (版本 v0.42.0-sakura-6 Windows amd64) 后，提示同样的错误。

已验证从官网下载的压缩包的 md5 值为 `124d11cc00c5e5971936e5264cfb5dff`，与下载页面比对无误。解压后的 frpc.exe 的 md5 值为 `4a2e45b55e966f0d3a8bcb48a4e1ef22`。

请问这是官网的 frpc.exe 文件有问题，还是其他的原因引起的？

**Reply by Lurito** [reply]
了解了，谢谢！

[a comment by an automated account is not shown]
